edb/pgsql/parser/ast_builder.py

```diff
@@ -46,8 +46,8 @@ def _maybe(
     return None
 
 
-def _ident(t: T) -> U:
```

Comment:
I'd drop the `T` and just make it `object` or `Any`. Not much point in having a type param only used once.